### PR TITLE
created new POM class

### PR DESCRIPTION
**Pull request by Boykareturns**
----------

### CODE REVIEW

File: ComcastCRMGUIFramework/src/main/java/com/comcast/crm/objectrepositoryutility/CreateProductPage.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package com.comcast.crm.objectrepositoryutility;
+
+public class CreateProductPage {
+
+	
+	//code1 
+	//code2
```
Comment (Boykareturns) [inline ComcastCRMGUIFramework/src/main/java/com/comcast/crm/objectrepositoryutility/CreateProductPage.java:7]: change the Xpath
Dont harcode the data

**Review by Boykareturns — COMMENTED** [review]
added few comments

File: ComcastCRMGUIFramework/src/main/java/com/comcast/crm/objectrepositoryutility/CreateProductPage.java (hunk 1 above)
Comment (MohitAutomationEngrr) [inline ComcastCRMGUIFramework/src/main/java/com/comcast/crm/objectrepositoryutility/CreateProductPage.java:7]: Change the Xpath
Do not harcode the data

**Review by MohitAutomationEngrr — COMMENTED** [review]
added few comments

File: ComcastCRMGUIFramework/src/main/java/com/comcast/crm/objectrepositoryutility/CreateProductPage.java (hunk 1 above)
Comment (Boykareturns) [reply ComcastCRMGUIFramework/src/main/java/com/comcast/crm/objectrepositoryutility/CreateProductPage.java:7]: fixed

**Comment by MohitAutomationEngrr** [comment]
merged.

**Pull request merged**